skia/RSkComponentRNSVGUse.cpp

```diff
@@ -80,14 +81,18 @@ bool RSkComponentRNSVGUse::onPrepareToRender(SkSVGRenderContext* ctx) const {
 }
 
 void RSkComponentRNSVGUse::onRender(const SkSVGRenderContext& ctx) const {
-
-  const auto ref = ctx.findNodeById(href_);
-  if (!ref) {
-    RNS_LOG_ERROR(" !!! Id not Found !!!!");
-    return;
+  if(rootNode_) {
+    auto rootSvgContainerNode=static_cast<RSkComponentRNSVGSvgView *>(rootNode_);
+    if(rootSvgContainerNode) {
+       RSkSVGNode**  nodeRef = rootSvgContainerNode->rskNodeIDMapper.find(href_);
+      if (!nodeRef && !(*nodeRef)) {
```

Comment:
This should be || (OR) right. If nodeRef is null then second condition will crash.

Reply:
Yes it should be ||. My Bad.